README.md

```diff
@@ -9,13 +9,59 @@ git submodule update
 # Install tcmalloc according to
 # https://github.com/gperftools/gperftools/blob/master/INSTALL
 
-pushd redis && env USE_TCMALLOC=yes make -j && popd
-pushd glog && cmake . && make -j install && popd
-pushd leveldb && make -j && popd
+cd redis
+env USE_TCMALLOC=yes make -j
+cd ..
+
+cd glog 
+cmake . 
+make -j install 
+cd ..
+
+cd leveldb 
+make -j 
+cd ..
+
+# Install grpc. A suitable version is included as a submodule of etcd3 but you
+# can install the latest version separately on the system too.
+cd etcd3
+git submodule update --init
```

Comment:
After this step I'm seeing
```
credis/etcd3(grayson-master*) » git submodule update --init
Submodule 'etcd3-cpp' (https://github.com/graysonchao/etcd3-cpp) registered for path './'
Submodule 'nlohmann' (https://github.com/nlohmann/json) registered for path '../nlohmann'
Cloning into '/Users/zongheng/Dropbox/workspace/riselab/credis/etcd3'...
Cloning into '/Users/zongheng/Dropbox/workspace/riselab/credis/nlohmann'...
Submodule path './': checked out 'ebcfd5c884dab2132c7b49a008fff6299d92f859'
Submodule path '../nlohmann': checked out '4efa8cdb4c7ae03674da7a29e1d70ae067836643'
error: Server does not allow request for unadvertised object ef9e360d94275419eb4d923dbbf284c2243f04ce
Fetched in submodule path '../redis', but it did not contain ef9e360d94275419eb4d923dbbf284c2243f04ce. Direct fetching of that commit failed.
```
Seems harmless (I can proceed to the rest of the commands) but good to fix

Reply:
Hmm. That's not good - you need commit `ef9e36` because it's where I added a couple of accessor methods to the Redis server struct (https://github.com/graysonchao/redis/commit/ef9e360d94275419eb4d923dbbf284c2243f04ce).

Once https://github.com/pcmoritz/redis/pull/1 is merged, this error will go away.

For now, you can fix it by running this:
```
git config submodule.redis.url https://github.com/graysonchao/redis
```

And it can be switched back using the same command but with `/pcmoritz/redis`.